example/lib/apis/user_apis.dart

```diff
@@ -5,6 +5,11 @@ callUpdateProfileApi(String name, String mobile) {
       name, mobile, (bool isUpdated) {}, (ApiErrorModel error) {});
 }
 
+updateNotificationSettingApi(bool notification) {
```

Comment:
can we rename the notification variable to shouldEnableNotification

Reply:
done